ldm/models/diffusion/cross_attention_control.py

```diff
@@ -14,7 +14,7 @@
 
 from compel.cross_attention_control import Arguments
 from diffusers.models.unet_2d_condition import UNet2DConditionModel
-from diffusers.models.cross_attention import AttnProcessor
```

Comment:
Did this break? We tried to deprecate the naming so it should have still worked but thrown a warning - maybe we messed up something

Reply:
I see this still works, but:
```py
from diffusers.models.attention import CrossAttention
```

stopped working